src/librustdoc/clean/types.rs

```diff
@@ -75,6 +75,14 @@ impl ItemId {
         }
     }
 
+    #[inline]
+    crate fn is_local_impl(self) -> bool {
+        match self {
+            ItemId::Blanket { impl_id, .. } => impl_id.is_local(),
+            ItemId::Auto { .. } | ItemId::DefId(_) | ItemId::Primitive(_, _) => false,
+        }
+    }
```

Comment:
This function's name is misleading. `ItemId::DefId` can represent (non-blanket) impls, and likewise, `ItemId::Auto` represents auto-trait impls.

I'd prefer just inlining the function into the call-site since it's only used once and doesn't seem that generally applicable.